src/mango/test/13-users-db-find-test.py

```diff
@@ -12,63 +12,65 @@
 # the License.
 
 
-import mango, requests
+import mango, requests, unittest
 
 
-class UsersDbFindTests(mango.UsersDbTests):
-    def test_simple_find(self):
-        docs = self.db.find({"name": {"$eq": "demo02"}})
-        assert len(docs) == 1
-        assert docs[0]["_id"] == "org.couchdb.user:demo02"
-
-    def test_multi_cond_and(self):
-        self.db.create_index(["type", "roles"])
-        docs = self.db.find({"type": "user", "roles": {"$eq": ["reader"]}})
-        assert len(docs) == 1
-        assert docs[0]["_id"] == "org.couchdb.user:demo02"
-
-    def test_multi_cond_or(self):
-        docs = self.db.find(
-            {"$and": [{"type": "user"}, {"$or": [{"order": 1}, {"order": 3}]}]}
-        )
-        assert len(docs) == 2
-        assert docs[0]["_id"] == "org.couchdb.user:demo01"
-        assert docs[1]["_id"] == "org.couchdb.user:demo03"
-
-    def test_sort(self):
-        self.db.create_index(["order", "name"])
-        selector = {"name": {"$gt": "demo01"}}
-        docs1 = self.db.find(selector, sort=[{"order": "asc"}])
-        docs2 = list(sorted(docs1, key=lambda d: d["order"]))
-        assert docs1 is not docs2 and docs1 == docs2
-
-        docs1 = self.db.find(selector, sort=[{"order": "desc"}])
-        docs2 = list(reversed(sorted(docs1, key=lambda d: d["order"])))
-        assert docs1 is not docs2 and docs1 == docs2
-
-    def test_fields(self):
-        selector = {"name": {"$eq": "demo02"}}
-        docs = self.db.find(selector, fields=["name", "order"])
-        assert len(docs) == 1
-        assert sorted(docs[0].keys()) == ["name", "order"]
-
-    def test_empty(self):
-        docs = self.db.find({})
-        assert len(docs) == 3
-
-
-class UsersDbIndexFindTests(UsersDbFindTests):
-    def setUp(self):
-        self.db.create_index(["name"])
-
-    def test_multi_cond_and(self):
-        self.db.create_index(["type", "roles"])
-        super(UsersDbIndexFindTests, self).test_multi_cond_and()
-
-    def test_multi_cond_or(self):
-        self.db.create_index(["type", "order"])
-        super(UsersDbIndexFindTests, self).test_multi_cond_or()
-
-    def test_sort(self):
-        self.db.create_index(["order", "name"])
-        super(UsersDbIndexFindTests, self).test_sort()
+# @unittest.skip("this FDB doesn't support this")
```

Comment:
What's not supported? These all look like normal things that should work?

Reply:
Ah. Lets uncomment but leave the skip with a message to the effect of "Re-enable once the `_users` db is implemented".

Reply:
I have to leave them commented out. Nose doesn't skip classes https://github.com/nose-devs/nose/issues/946

Reply:
That link says you can skip from a setupUpClass function:

https://github.com/nose-devs/nose/issues/946#issuecomment-520127254

Leaving them commented out means we're more likely to forget about them is all. I'd prefer to see a thing in our test logs that say we're skipping something rather than just silently ignoring it.